mc-sema/cfgToLLVM/raiseX86_64.cpp

```diff
@@ -273,102 +277,58 @@ bool addEntryPointDriver(Module *M,
             aliasMCSemaScope(new StoreInst(curArg, arg4, driverBB));
         }
     } else if (getSystemOS(M) == llvm::Triple::Linux) {
-        //#else
-        if(fwd_it != fwd_end) {
-            int   k = x86_64::getRegisterOffset(RDI);
-            Value *rdiFieldGEPV[] = {
-                CONST_V<64>(driverBB, 0),
-                CONST_V<32>(driverBB, k)
+      //#else
+      unsigned fp_reg_num = 0;
+      unsigned notfp_reg_num = 0;
+      Value *args_fp[8];
```

Comment:
Add some constants here, e.g.
```C++
enum : size_t {
  kNumFPRegs = 8,
  kNumIntRegs = 6
};
```
and use these below instead of the literal constants `6` and `8`.